app/util/data_preparation/confluence/prepare-data.py

```diff
@@ -90,6 +90,13 @@ def write_test_data_to_files(dataset):
     __write_to_file(CONFLUENCE_USERS, users)
 
 
+def __check_default_confluence_language(confluence_api):
+    confluence_language = confluence_api.get_locale()
```

Comment:
All this code is situated in the `confluence` directory and all these methods apply just to Confluence. So using `confluence` in the variables naming is redundant. `current_language` is better.

Reply:
done